vaadin-archetype-liferay-portlet/src/main/resources/archetype-resources/src/main/webapp/WEB-INF/liferay-portlet.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<!DOCTYPE liferay-portlet-app PUBLIC "-//Liferay//DTD Portlet Application 7.1.0//EN" "http://www.liferay.com/dtd/liferay-portlet-app_7_1_0.dtd">
+
+<liferay-portlet-app>
+    <portlet>
+        <portlet-name>${portletName}</portlet-name>
+        <instanceable>false</instanceable>
+        <css-class-wrapper>${portletName}-css</css-class-wrapper>
```

Comment:
The only concern using `portletName` is that if it contains spaces we may have an invalid css class name.
I can't find in the specifications any restriction on portlet name, but all sample code uses camel case identifier, so it could be ok to use `portletName`.

Reply:
Removed